config/clusters/smithsonian/common.values.yaml

```diff
@@ -97,7 +97,17 @@ basehub:
                   default: true
                   slug: geospatial
                   kubespawner_override:
-                    image: rocker/geospatial:4.3
+                    image: rocker/binder:4.3
```

Comment:
```suggestion
                    # rocker/binder builds on top of rocker/geospatial and will
                    # for releases after 4.3.1 include jupyterhub as required to
                    # be compatible with our jupyterhub.
                    #
                    # FIXME: Replace quay.io/2i2c/rocker-binder with
                    #        rocker/binder as soon as a version beyond 4.3.1 is
                    #        released or https://github.com/rocker-org/rocker-versioned2/pull/671 resolves it.
                    #
                    image: quay.io/2i2c/rocker-binder:4.3.1
```

Reply:
@consideRatio this is not necessary, as the image was rebuilt and pushed to contain jupyterhub now https://github.com/rocker-org/rocker-versioned2/pull/668#issuecomment-1610066198